nixos/modules/services/web-apps/phylactery.nix

```
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.phylactery;
+in {
+  options.services.phylactery = {
+    enable = mkEnableOption "Whether to enable Phylactery server.";
+
+    host = mkOption {
+      type = types.str;
+      default = "";
```

Either remove the default or make the type `types.nullOr types.str` and make the default null.

---

The implementation allows an empty string for host, e.g. :8080 is considered a valid address.  If localhost is better, feel free to tell me to change.

---

`localhost` would be clearer.
```suggestion
      type = types.str;
      default = "localhost";
```

---

Applied!